### ISSUE TITLE
Timecodes for "ML Zoomcamp 8.13 - Summary"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#148]
Youtube video: https://www.youtube.com/watch?v=mn0BcXJlRFM

**Comment by dimzachar** [comment]
0:00:00 - Summary of machine learning session using CNN
0:00:46 - Using pre-trained neural networks in Keras.
0:01:28 - Building neural network and saving models.
0:02:10 - Adding dense layers, regularization, augmentation, training.
0:02:58 - Training model evaluation and image prediction.
0:03:40 - Augmentation libraries for deep learning frameworks.
0:04:32 - Explore different models for speed and accuracy.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks
